Source/ASCollectionView.mm

```diff
@@ -367,6 +367,7 @@ - (void)scrollToItemAtIndexPath:(NSIndexPath *)indexPath atScrollPosition:(UICol
 
 - (void)relayoutItems
 {
+  [self invalidateFlowLayoutDelegateMetrics];
```

Comment:
@appleguy Now that #431 is merged, please update this PR. Also, should `[self invalidateFlowLayoutDelegateMetrics];` be called inside the invalidation block?